www/_template/guides/connecting-tools.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: layouts/guide.njk
+title: The Snowpack Guide to connecting your favorite tools
+description: 'How do you use your favorite tools in Snowpack? This Guide will help you get started'
+tags: guides
+sidebarTitle: Connecting your favorite tools
+---
+
+One of the biggest questions we get in our discussion forums and Discord is how to connect tools to Snowpack. EsLint, PostCSS, SASS, and any other of the useful tools developers use to build JavaScript these days. In this Guide we'll go over how to use your favorite tools with Snowpack. Almost all work with just a few additions to a configuration file.
```

Comment:
I (very egotistically) always try to avoid documentation that implies issues / difficulty of anything, which this first sentence does trigger for me a bit. What about:

One of the most common questions we get is "How do I connect my favorite tool to Snowpack?" In this guide we'll go over the three different ways that you can integrate third-party tooling into your Snowpack dev environment or build pipeline:

- Snowpack plugin
- Integrated CLI script (via `@snowpack/plugin-run-script`)
- Run separately, outside of Snowpack (ex: in your `package.json`)

Then, no need for the "three ways" section below